waggle-dance-api/src/test/java/com/hotels/bdp/waggledance/api/model/MetastoreTunnelTest.java

```diff
@@ -49,6 +50,15 @@ public void typical() {
     assertThat(violations.size(), is(0));
   }
 
+  @Test
+  public void infiniteTimeout() {
```

Comment:
We should document in the README that setting it to zero means no timeout.

Reply:
will document